proposals/0029-cooperative-vector.md

```diff
@@ -391,23 +391,23 @@ enum class ComponentType : uint32_t {
   U32, // = 5
   I64,
   U64,
-  F16, // = 7
-  F32, // = 8
+  F16, // = 8
+  F32, // = 9
   F64,
   SNormF16,
   UNormF16,
   SNormF32,
   UNormF32,
   SNormF64,
   UNormF64,
-  PackedS8x32, // = 16
-  PackedU8x32, // = 17
+  PackedS8x32, // = 17
+  PackedU8x32, // = 18
 
   // BEGIN NEW FOR SM 6.9
-  U8,      // = 18
-  I8,      // = 19
-  F8_E4M3, // = 20  
-  F8_E5M2, // = 21
```

Comment:
The matching API enum below needs to be fixed to match.

Reply:
Good catch, missed that.